package.json

```diff
@@ -3,6 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@astar-network/astar-api": "^0.1.12",
+    "@polkadot/api": "^9.5.1",
```

Comment:
Does it make sense to export polkadot/api from astar.js lib so users can omit this import and use the same polkadot.js version everywhere in the project?